<launch>
    <node pkg="teleop_controller" name="teleopController" type="teleopController"/>
    <node pkg="joy" name="joy" type="joy_node"/>
    <node pkg="dynamic_reconfigure" name="dynamic_reconfigure" type="reconfigure_gui"/>
</launch>
